main.py

```diff
@@ -1,7 +1,33 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Actor, Genre
 
 
 def main() -> QuerySet:
-    pass
+    genres = ("Western", "Action", "Dramma")
+    for genre in genres:
+        Genre.objects.create(name=genre)
+    actors = (
+        ("George", "Klooney"),
+        ("Kianu", "Reaves"),
+        ("Scarlett", "Keegan"),
+        ("Will", "Smith"),
+        ("Jaden", "Smith"),
+        ("Scarlett", "Johansson")
+    )
+    for first_name_, last_name_ in actors:
```

Comment:
Why you need _ at the end of variable names? I do not see any other first_name and last_name